core/templates/dev/head/pages/exploration_editor/exploration_save_and_publish_buttons_directive.html

```diff
@@ -1,7 +1,9 @@
 <ul class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak">
   <li ng-if="showPublishButton()">
     <div class="btn-group" style="margin-right: 10px; margin-top: 8px;"
-         title="<[getPublishExplorationButtonTooltip()]>">
+         uib-tooltip="<[getPublishExplorationButtonTooltip()]>"
+         tooltip-placement="bottom"
+         ng-class="{'div-disable': isExplorationLockedForEditing() || countWarnings()}" ng-click="showTooltip()">
```

Comment:
is div-disable a standard class you want to use? If not, could you please find a more descriptive name. Maybe something like disable-publish-button or likewise.

Reply:
I need to use this class both for publish and save draft buttons when they are disabled. Would it be better to use separate names for both?

Reply:
I think separate names would be better.

Reply:
Done.